### ISSUE TITLE
Add information to the MongoDB

### ISSUE THREAD

**Issue by DenDix** [meanjs/mean#1906]
I just want to add and access information to/from the mongodb. I'm trying to set up a simple blog in mean.js 0.5.0. What are the basics? I can't seem to find the information anywhere. I think I need to add lines to express.js, but exactly what is the code I should use and where do I put it? There must be a working example somewhere of using the database on mean.js 0.5.0.

**Comment by s0j0hn** [comment]
You will need to setup mongodb server. By default meanjs try to access a local mongodb server. Meanjs will  seed automaticly your database

**Comment by DenDix** [comment]
It seems like I already have a mongodb server set up, which I think I needed to do in order to install mean.js 0.5.0. If at the terminal I type "mongo", then "show dbs", I get 3 lines "local", "mean-dev", "mean-test". If I have the terminology correct these are all "collections" on my mongodb. I want to set up a new "collection" and add plus retrieve data on it. 

To create a new collection maybe I execute a command like
 "var PostModel=mongoose.model("PostModel",PostScheme);"

But where would I put this line, possibly somewhere in "express.js"?

**Comment by s0j0hn** [comment]
No, you need to create a new model then you can make your crud request to mongodb, just like articles

**Comment by mleanos** [comment]
> f at the terminal I type "mongo", then "show dbs", I get 3 lines "local", "mean-dev", "mean-test".

Those are databases. Within those databases you will find your collections.

Just as @s0j0hn hinted at, just use the Articles module as an example. Articles will show you basic CRUD operations, server-side routes & policies, along with all the necessary client-side implementations. The reason for the Articles module is for an example.

**Comment by DenDix** [comment]
My problem was I didn't realize that the hbs view engine wasn't working when I first installed meanjs 0.5.0. Since I had a blank screen I put all my files in /public and have been working from scratch with a static html. I did in fact access the mongodb this way by creating my own module inside express.js. But clearly, when the view engine is running, it would be much easier to just work from the "articles" example. Until I realized this, the responses I was getting from my question on the forum did seem confusing.
Here is the error I am getting in the Chrome browser when running meanjs without accessing my static html:
Uncaught ReferenceError: angular is not defined
    at config.js:25
    at config.js:36
init.js:5 Uncaught ReferenceError: angular is not defined
    at init.js:5
    at init.js:54
(and so on )

I find this a little odd, since angular isn't mentioned on those lines in config.js.This is the next issue to try and solve which I haven't actually looked into yet.

Thanks for everyone's help. I'll post something when hopefully I figure out what's going wrong.
